### ISSUE TITLE
aria-hidden and hidden attribute updates needed

### ISSUE THREAD

**Issue by scottaohara** [ThePacielloGroup/WAI-ARIA-Usage#29]
[test case](https://w3c.github.io/html-aria/tests/aria-hidden-invalid-element-test.html)
expected results: all elements in this testcase should fail, as none of them are allowed aria-* attributes, including aria-hidden.

[test case](https://w3c.github.io/html-aria/tests/aria-hidden-and-hidden.html)
expected results: all elements in this testcase would fail as they all have both `aria-hidden` and the `hidden` attribute

**Comment by gezlemon** [comment]
Added tests for aria-hidden usage

**Comment by scottaohara** [comment]
everything checks out but for `base` from test one is not indicated as an element which should not have `aria-hidden` specified to it. 

for all intents and purposes, this is a good enough return for me to mark this as a pass for aria in html, but just thought i'd mention that

**Comment by gezlemon** [comment]
Thanks, Scott. Fixed a bug in the logic that was missing base with aria-hidden.

**Comment by scottaohara** [comment]
thanks!
